tasks.py

```diff
@@ -614,6 +626,28 @@ def vpc_components(ctx):
     return subnet_id, sg_id
 
 
+def oc_base_url(ctx):
+    cmd = ("aws {} opsworks --region us-east-1 describe-stacks "
+           "--query \"Stacks[?Name=='{}'].StackId\" --output text")\
+        .format(profile_arg(), OC_CLUSTER_NAME)
```

Comment:
Where does `OC_CLUSTER_NAME` come from?

Reply:
New environment variable that would replace VPC_ID and OPENCAST_BASE_URL